include/galsim/hsm/PSFCorr.h

```diff
@@ -122,6 +122,10 @@ namespace hsm {
      *                           KSB method of PSF correction.  Warning: deviating from default
      *                           value of 2 results in code running more slowly, and results have
      *                           not been significantly tested.
+     * @param convergence_threshold Accuracy (in x0, y0, and sigma as a fraction of sigma) when
+     *                           calculating adaptive moments. The value of sigma used for the
+     *                           convergence criterion is the minimum of the "guessed" value and
+     *                           the "current" value.
```

Comment:
I don't think the second sentence is always true.  It's only true if ```std::abs(dsigma)>convergence_factor```.

It also seems like perhaps more information than is necessary, in any case?